packages/frontend/src/components/common/Filters/configs.ts

```diff
@@ -192,7 +204,7 @@ export const getFilterRuleLabel = (
 };
 
 export const getFilterRuleTables = (
-    filterRule: FilterRule | DashboardFilterRule,
+    filterRule: ConditionalRule,
```

Comment:
`DashboardFilterRule` extends `FilterRule` and `FilterRule` extends `ConditionalRule`